packages/composables/src/appContext.ts

```diff
@@ -3,13 +3,28 @@ import { ShopwareApiInstance } from "@shopware-pwa/shopware-6-client";
 import { VueConstructor } from "vue";
 
 /**
- * Applicatoin Context for Shopware PWA. It's an extended Vue instance.
+ * @beta
+ */
+export interface Routing {
```

Comment:
probably needs to be regenerated locally, as files above show warning related to this interface